ext/date/tests/bug79015.phpt

```diff
@@ -0,0 +1,9 @@
+--TEST--
+Bug #79015 (undefined-behavior in php_date.c)
+--FILE--
+<?php
+var_dump(unserialize('O:12:"DateInterval":1:1s:1:"f";i:9999999999990;'));
```

Comment:
Please fix the end of the payload so the var_dump works.

Reply:
Makes sense. Thanks! The diff without the changes in php_date.c would be:
````
015+   float(-9223372036854.8)
015-   float(-1)
````